ikasaneip/sample/spring-boot/jms/src/test/java/com/ikasan/sample/spring/boot/builderpattern/JmsSampleFlowTest.java

```diff
@@ -281,20 +313,21 @@ public void test_exclusion_followed_by_resubmission()
         // start the flow and assert it runs
         flowTestRule.startFlow();
 
-        with().pollInterval(500, TimeUnit.MILLISECONDS).and().await().atMost(60, TimeUnit.SECONDS)
-              .untilAsserted(() -> flowTestRule.assertIsSatisfied());
+        with().pollInterval(50, TimeUnit.MILLISECONDS).and().await().atMost(60, TimeUnit.SECONDS)
```

Comment:
Could we add some nice method into IkasanFlowTestRule 

flowTestRule.assertIsSatisfied(maxWaitTimeInSeconds)

Reply:
can do - the puprose of this work was to provide the primitive actions and utilities to get our tests running without reloading the context. And use the JmsSampleFlowTest to prove this. Didnt want to provide a final test framework as part of this work